README.rst

```diff
@@ -94,6 +94,40 @@ Look at the new Repository Version created
   }
 
 
+Upload a Role to Pulp
+---------------------
+
+Download a role version.
+
+``curl -L https://github.com/pulp/ansible-pulp3/archive/master.tar.gz -o pulp.tar.gz``
+
+Create an Artifact by uploading the role version tarball to Pulp.
+
+``$ export ARTIFACT_HREF=$(http --form POST http://localhost:8000/api/v3/artifacts/ file@pulp.tar.gz | jq -r '._href')``
+
+
+Create a Role content unit
+--------------------------
+
+Create an Ansible role in Pulp.
+
+``$ export ROLE_HREF=$(http http://localhost:8000/api/v3/content/ansible/roles/ namespace=pulp name=pulp | jq -r '._href')``
+
+
+Create a ``role version`` from the Role and Artifact
+-----------------------------------------------------
+
+Create a content unit and point it to your Artifact and Role
+
+``$ export CONTENT_HREF=$(http POST ${ROLE_HREF}versions/ version=0.0.1 artifact=$ARTIFACT_HREF | jq -r '._href')``
+
+
+Add content to repository ``foo``
+---------------------------------
+
+``$ http POST $REPO_HREF'versions/' add_content_units:="[\"$CONTENT_HREF\"]"``
```

Comment:
I had trouble with this command. It says:

```
(pulp) [vagrant@pulp3 ~]$ http POST $REPO_HREF'versions/' add_content_units:="[\"$CONTENT_HREF\"]"
HTTP/1.1 400 Bad Request
Allow: GET, POST, HEAD, OPTIONS
Content-Type: application/json
Date: Mon, 23 Apr 2018 20:00:09 GMT
Server: WSGIServer/0.2 CPython/3.6.2
Vary: Accept

[
    "URI http://localhost:8000/api/v3/content/ansible/roles/069d5cf7-bc77-46b7-a2cf-95b390f03548/versions/3f7bacb8-4b62-4208-bbeb-e74226bf3984/ is not a valid content."
]
```

Reply:
Here's the fix:

https://github.com/pulp/pulp/pull/3468

Reply:
That other PR got merged so I resumed my testing.